### ISSUE TITLE
check_ntp_peer truechimers option

### ISSUE THREAD

**Issue by jvandermeulen** [nagios-plugins/nagios-plugins#435]
check_ntp_peer v2.2.1 uses this example for truechimers

>  Only check the number of usable time sources ("truechimers"):
>   ./check_ntp_peer -H ntpserv -m @5 -n @3

Our timeserver has 4 usable time sources. So what I would normally do to check if it drops below 4 (warning) or even drops below 2 (critical):

```
./check_ntp_peer -H timeserver02 -m 4: -n 2:
NTP OK: Offset -0.000252 secs, truechimers=4|offset=-0.000252s;60.000000;120.000000; truechimers=4;0;0;0
```
![2019-02-26 16_06-ccm nagios xi](https://user-images.githubusercontent.com/838001/53422852-8fd3f100-39e0-11e9-852f-9cc5821dfb08.png)

If  I use the example on the help page (using a "range" with @-sign and just one integer) it gives a warning:
```
 ./check_ntp_peer -H timeserver02 -m @4 -n @2
NTP WARNING: Offset -1.1e-05 secs, truechimers=4 (WARNING)|offset=-0.000011s;60.000000;120.000000; truechimers=4;4;2;0
```

So I'll stick to the colon mentioned in the [Nagios Development Guidelines](https://nagios-plugins.org/doc/guidelines.html#THRESHOLDFORMAT) instead of trying to understand the `-m @5 -n @3` example of `./check_ntp_peer -h`

In both cases the performance data returned is not what I expected.

See also [this](https://support.nagios.com/forum/viewtopic.php?f=16&t=47744) thread

**Comment by Madlohe** [comment]
As mentioned in the thread, '@' syntax includes the boundaries when it inverts the range, so `-m 4: -n 2:` will be equivalent to `-m @3 -n @1`.

I agree that the performance data is wrong, though. Thanks for reporting this.

**Comment by sawolf** [comment]
Hi @jvandermeulen,

I've set up a PR which should fix the performance data issues - #488. If you'd like to verify that these changes work in your environment, please compile from there and let me know how it works.
